app/serializers/credentials_serializer.rb

```diff
@@ -3,40 +3,40 @@ def initialize(user: , application:)
     @user, @application = user, application
   end
 
-  def serialize
-    {
-      user: serialized_user,
-      profile: serialized_profile,
-      roles: serialized_roles,
-    }
-  end
-
-  def to_json(opts = {})
-    serialize.to_json opts
+  def as_json(opts = {})
+    serialize
   end
 
   private
 
   attr_reader :user, :application
 
+  def serialize
```

Comment:
Can we make this method (CredentialsSerializer#serialize) public please (as discussed on a previous PR)